default.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="origin" fetch="https://git.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git" />
+  <default revision="master" remote="origin"  sync-j="4" />
```

Comment:
Is this a Canonical-ism? I find it a little irregular that we don't treat master as a "current stable" branch and use branches for features. While I completely understand the point of a release branch, it seems strange to me that we should use any branching strategy that doesn't trend toward a canonical (lowercase c) master.